_assets/stylesheets/pages/_what-we-believe.scss

```diff
@@ -0,0 +1,4 @@
+.what-we-believe-bg {
+  background-image: url("//crds-media.imgix.net//6oN4t6m8wvoZBH64lLMfbM/0d7b34596c93676ad7942df66debf57d/what-we-believe-jumbotron.png");
+  height: 455px;
```

Comment:
Do you need to set this height? This could be done by using classes like `soft-ends` inside of the jumbotron.